car.capnp

```diff
@@ -116,6 +116,7 @@ struct CarEvent @0x9b1657f34caf3ad3 {
     resumeBlocked @113;
     steerTimeLimit @115;
     vehicleSensorsInvalid @116;
+    adasFaulted @118;
```

Comment:
it's not just adas faulted, it's anything faulted that doesn't go in accFaulted or steer fault

Reply:
how about `carFaultedNonCritical`? I want it to be clear without a comment

Reply:
since we are choosing to not make it a noEntry, I'll leave it with this prefix. if there is another type of fault that is not acc or lkas in the future, but *does* disallow disengagement, we can just add `carFaulted`